examples/lighting-app/linux/BUILD.gn

```diff
@@ -57,6 +57,11 @@ executable("chip-lighting-app") {
       "Rpc.cpp",
     ]
 
+    include_dirs += [
```

Comment:
Please do not directly add include_dirs for your dependencies.

Reply:
Done